addons/mass_mailing/static/src/iframe/mass_mailing_iframe.js

```diff
@@ -224,6 +229,20 @@ export class MassMailingIframe extends Component {
         this.state.ready = true;
     }
 
+    checkAllInline(el) {
+        // would be nice to move this to a readonly plugin once they're implemented
+        return [...el.children].every((child) => {
+            if (child.tagName === "T") {
+                return this.checkAllInline(child);
+            } else {
+                return (
+                    child.nodeType !== Node.ELEMENT_NODE ||
+                    this.window.getComputedStyle(child).display === "inline"
```

Comment:
`this.window` makes sense in the context of a Plugin, but here you should use `this.iframeRef.el.contentWindow` instead